index.html

```diff
@@ -52,5 +53,124 @@ <h1>
         <a href=""><img src="images/Disabled.svg" alt="" /></a>
       </div>
     </div>
+
+    <!-- The header for my recent works -->
+
+    <div class="top-tile">
+      <h1 class="card-title">
+        My Recent <br />
+        Works
+      </h1>
+      <hr />
+    </div>
+    <!-- The main card container -->
+    <div class="card-container">
+      <div class="card">
```

Comment:
- [ ] Kindly wrap your work container in a `<section>` tag for accessibility reasons